recipe/meta.yaml

```
@@ -17,24 +17,22 @@ source:
 build:
   string: cpu{{ CONDA_PY }}_h{{ PKG_HASH }}_{{ PKG_BUILDNUM }}  # [cuda_compiler_version == "None"]
   string: cuda{{ cuda_compiler_version | replace('.', '') }}_h{{ PKG_HASH }}_{{ PKG_BUILDNUM }}  # [cuda_compiler_version != "None"]
-  script:
-    - set CUTLASS_PATH=%cd%  # [win and cuda_compiler_version != "None"
-    - export CUTLASS_PATH=$(pwd)  # [unix and cuda_compiler_version != "None"]
-    - export CUDA_INSTALL_PATH=$(which nvcc | awk -F'/bin/nvcc' '{print $1}')  # [unix]
-    - {{ PYTHON }} build_lib.py
-    - {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation --no-cache-dir
-  number: 0
+  number: 1
   skip: true  # [osx or cuda_compiler_version == "11.2"]
-  skip: true  # [win and cuda_compiler_version != "None"]
 
 requirements:
   build:
     - {{ compiler('c') }}
     - {{ compiler('cxx') }}
     - {{ compiler('cuda') }}  # [cuda_compiler_version != "None"]
+    - {{ stdlib("c") }}
     - cmake
     - ninja
   host:
+    - cuda-cudart-dev    # [(cuda_compiler_version or "").startswith("12") and win]
+    - cuda-nvcc          # [(cuda_compiler_version or "").startswith("12") and win]
+    - cuda-nvrtc-dev     # [(cuda_compiler_version or "").startswith("12")]
+    - cuda-nvrtc-static  # [(cuda_compiler_version or "").startswith("12")]
```

```suggestion
    - cuda-nvrtc-static  # [(cuda_compiler_version or "").startswith("12")]
```

Are you sure that `cuda-nvrtc-static` is required? Ideally `cuda-nvrtc-dev` should be sufficient, but if the use of `cuda-nvrtc-static` is required by the project no problem.

---

I'm getting these two errors, so I believe that they do not get installed with `cuda-nvrtc-dev` (see https://conda-metadata-app.streamlit.app/?q=conda-forge%2Flinux-64%2Fcuda-nvrtc-dev-12.4.127-hd3aeb46_1.conda):

```sh
/home/conda/feedstock_root/build_artifacts/warp-lang_1715264392388/_build_env/bin/../lib/gcc/x86_64-conda-linux-gnu/12.3.0/../../../../x86_64-conda-linux-gnu/bin/ld: cannot find -lnvrtc_static: No such file or directory
/home/conda/feedstock_root/build_artifacts/warp-lang_1715264392388/_build_env/bin/../lib/gcc/x86_64-conda-linux-gnu/12.3.0/../../../../x86_64-conda-linux-gnu/bin/ld: cannot find -lnvrtc-builtins_static: No such file or directory
```

---

Apparently it is needed indeed:
~~~
/home/conda/feedstock_root/build_artifacts/warp-lang_1715264399929/_build_env/bin/../lib/gcc/x86_64-conda-linux-gnu/12.3.0/../../../../x86_64-conda-linux-gnu/bin/ld: cannot find -lnvrtc_static: No such file or directory
/home/conda/feedstock_root/build_artifacts/warp-lang_1715264399929/_build_env/bin/../lib/gcc/x86_64-conda-linux-gnu/12.3.0/../../../../x86_64-conda-linux-gnu/bin/ld: cannot find -lnvrtc-builtins_static: No such file or directory
~~~